docs/user/filesystem/sql.rst

```diff
@@ -79,7 +82,6 @@ have ingested GDELT data into S3 you can query it with SQL in the spark shell::
 
     // Select the top event codes
     spark.sql("SELECT eventCode, count(*) as count FROM gdelt " +
-              "WHERE dtg >= cast('2017-06-01T00:00:00.000' as timestamp) " +
-              "AND dtg <= cast('2017-06-30T00:00:00.000' as timestamp) " +
+              "WHERE dtg >= '2017-06-01T00:00:00Z' AND dtg <= '2017-06-30T00:00:00Z' " +
```

Comment:
you got rid of the cast?!

Reply:
yeah it's not needed any more - we fixed that a while ago. it still works with the cast but it seems cumbersome